release-notes/latest.md

```diff
@@ -40,3 +40,7 @@
 #### <sub><sup><a name="5410" href="#5410">:link:</a></sup></sub> feature
 
 * We've moved the "pin comment" field in the Resource view to the top of the page (next to the currently pinned version). The comment can be edited inline.
+
+#### <sub><sup><a name="5057" href="#5057">:link:</a></sup></sub> feature
+ 
+ * @HannesHasselbring and @tenjaa added a prometheus metric of the amount of tasks that are currently waiting to be scheduled when using the `limit-active-tasks` placement strategy
```

Comment:
Always nice to link the PR that provided the change! Also I think this should work with all the metrics emitters, not that I've exhaustively tested.
```suggestion
 * @HannesHasselbring and @tenjaa added a metric for the amount of tasks that are currently waiting to be scheduled when using the `limit-active-tasks` placement strategy. #5448
```